<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OduncTeslimIslemleri</class>
 <widget class="QDialog" name="OduncTeslimIslemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ödünç Alınan Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Teslim Edilen Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kitap_no">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_teslim">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Ödüncü Ver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ödünç Verme Tarihi</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="uye_no">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>130</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kitap No</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="teslim_tarihi">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Üye No</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="odunc_tablo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="teslim_tablo">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>80</y>
     <width>291</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
